<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassSelectDialog</class>
 <widget class="QDialog" name="ClassSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClassSelectDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton_class0">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>0号板</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_class1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>1号板</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_class2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>2号板</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_class3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>3号板</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
